README.md

```diff
@@ -63,11 +72,16 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   roslaunch godel_irb2400_support irb2400_blending_server.launch namespace:=[robot namespace]
   ```
   Client:
-  Cd into the 'src' directory of your catkin workspace and run the following:
+  Run following program:
```

Comment:
if for some reason all of your changes didn't make it you can try and do a ```git push``` again